src/platforms/react-native/manual-setup/sourcemaps.mdx

```diff
@@ -32,8 +36,9 @@ node_modules/@sentry/cli/bin/sentry-cli releases \
     files RELEASE_NAME \
     upload-sourcemaps \
     --dist DISTRIBUTION_NAME \
-    --strip-prefix /path/to/project/root \
-    /path/to/your/files
+    --url-prefix app:/// \
```

Comment:
```suggestion
   --strip-prefix /path/to/project/root
```

`url-prefix` is actually not needed but we need to keep `strip-prefix` to strip the prefix from the files.